bluesky/plan_stubs.py

```diff
@@ -1020,3 +1020,50 @@ def repeated_plan():
                     yield Msg('sleep', None, d)
 
     return (yield from repeated_plan())
+
+
+def update_epicsmotor_user_offset(epicsmotor, val):
+    '''Follows the correct procedure to update an EpicsMotor user offset.
+
+    There is a complex set of steps involved in correctly updating the user
+    offset value as the effect of changing its value depends on he values of
+    other signals, this plan_stub genreates the instructions to perform those
+    steps. See the section on Calibration-related fileds at https://epics.anl.gov/bcda/synApps/motor/R4-1/motorRecord.html>>>>>>>ccb1ced23ad740f7c7c0e9a5a312643f125c1599
```

Comment:
This url does to look right, assume 

```suggestion
    steps. See the section on Calibration-related fileds at https://epics.anl.gov/bcda/synApps/motor/R4-1/motorRecord.html
```